internal/dcache/cluster_manager/cluster_manager_impl.go

```diff
@@ -114,6 +128,81 @@ func (c *ClusterManagerImpl) ReportRVFull(rvName string) error {
 	return nil
 }
 
+func (cmi *ClusterManagerImpl) createClusterMapIfRequired(dCacheConfig *dcache.DCacheConfig, rvList []dcache.RawVolume) error {
+	isClusterMapExists, err := cmi.checkIfClusterMapExists(dCacheConfig.CacheId)
+	if err != nil {
+		log.Err("ClusterManagerImpl::createClusterMapIfRequired: Failed to check clusterMap file presence in Storage path %s error: %v", cmi.storagePath+"/ClusterMap.json", err)
+	}
+	if isClusterMapExists {
+		log.Trace("ClusterManager::createClusterMapIfRequired : ClusterMap.json already exists")
+		return nil
+	}
```

Comment:
we should log the clustermap json  found using %+v. That's a good debugging log.